api/hypercore/add_columnstore_policy.md

```diff
@@ -0,0 +1,108 @@
+
+   For example:
+
+   * 60 days after the data was added to the table:
+     ``` sql
+     CALL add_columnstore_policy('stocks_real_time', after => INTERVAL '60d');
+     ```
+   * 3 months prior to the moment you run the query:
+
+     ``` sql
+     CALL add_columnstore_policy('stocks_real_time', created_before => INTERVAL '3 months');
+     ```
+   * With an integer-based time column:
+
+     ``` sql
+     CALL add_columnstore_policy('table_with_bigint_time', BIGINT '600000');
+     ```
+   * Older than eight weeks:
+
+     ``` sql
+     CALL add_columnstore_policy('cpu_weekly', INTERVAL '8 weeks');
+     ```
+     
+1. **View the policies that you set or the policies that already exist** 
+
+   ``` sql
+   SELECT * FROM timescaledb_information.jobs
+   WHERE proc_name='policy_compression';
+   ```
+   See [timescaledb_information.jobs][informational-views].
+
+</Procedure>
+
+## Arguments
+
+Calls to `add_columnstore_policy` require either `after` or `created_before`, but cannot have both.
+
+<!-- vale Google.Acronyms = NO -->
+<!-- vale Vale.Spelling = NO -->
+
+| Name | Type | Default | Required | Description |
+|--|--|--|--|--|
+| `hypertable`              |REGCLASS| - | ✔ | Name of the hypertable or continuous aggregate to run this [job][job] on.|
+| `after`          |INTERVAL or INTEGER|- | ✖ | Add chunks containing data older than `now - {after}::interval` to the columnstore. <br/> Use an object type that matchs the time column type in `hypertable`: <ul><li><b><code>TIMESTAMP</code>, <code>TIMESTAMPTZ</code>, or <code>DATE</code></b>: use an <code>INTERVAL</code> type.</li><li><b> Integer-based timestamps </b>: set an integer type using the [integer_now_func][set_integer_now_func].</li></ul> `after` is mutually exclusive with `created_before`. |
+| `created_before` |INTERVAL| NULL | ✖ | Add chunks with a creation time of `now() - created_before` to the columnstore. <br/> `created_before` is <ul><li>Not supported for continuous aggregates.</li><li>Mutually exclusive with `after`.</li></ul> |
+| `schedule_interval`       |INTERVAL| 12 hours when [chunk_time_interval][chunk_time_interval] >= `1 day` for `hypertable`. Otherwise `chunk_time_interval` / `2`. | ✖        | Set the interval between the finish time of the last execution of this policy and the next start.|
+| `initial_start`     |TIMESTAMPTZ| The interval from the finish time of the last execution to the [next_start][next-start].| ✖| Set the time this job is first run. This is also the time that `next_start` is calculated from.|
+| `timezone`          |TEXT| UTC. However, daylight savings time(DST) changes may shift this alignment. | ✖ | Set to a valid time zone to mitigate DST shifting. If `initial_start` is set, subsequent executions of this policy are aligned on `initial_start`.|
+| `if_not_exists`     |BOOLEAN| `false` | ✖ | Set to `true` so this job fails with a warning rather than an error if a columnstore policy already exists on `hypertable` |
```

Comment:
This is not required
```suggestion
| `if_not_exists`     |BOOLEAN| `false` |  | Set to `true` so this job fails with a warning rather than an error if a columnstore policy already exists on `hypertable` |
```

Reply:
✔ indicated required, ✖  not required.

Reply:
Ah, I thought it was "crossed" and "not-crossed". Sorry about that.